components/dot_menu/index.ts

```diff
@@ -124,7 +124,6 @@ function mapStateToProps(state: GlobalState, ownProps: Props) {
         isCollapsedThreadsEnabled: collapsedThreads,
         threadReplyCount,
         isMobileView: getIsMobileView(state),
-        ...ownProps,
```

Comment:
There is no need to add `ownProps` to the return object from `mapStateToProps`
From https://react-redux.js.org/using-react-redux/connect-mapstate#:~:text=ownProps%20(optional)%E2%80%8B&text=You%20do%20not%20need%20to,a%20final%20set%20of%20props.
<img width="964" alt="image" src="https://user-images.githubusercontent.com/7575921/160854661-8f895a45-a8ab-4967-8696-0935e25badd1.png">